trigger/process.go

```diff
@@ -50,10 +50,10 @@ func NotificationsFromEvent(ctx context.Context, e *event.EventLogEntry) ([]noti
 		grip.Error(message.WrapError(err, msg))
 		return nil, err
 	}
-	grip.Info(msg)
 	if len(subscriptions) == 0 {
 		return nil, nil
 	}
+	grip.Info(msg)
```

Comment:
Changed these to conditionally log only if there's something to actually process. It seems like the majority of Splunk logs are for no-op cases (no subscription, no notification) rather than actually having work to do.